lms/djangoapps/courseware/features/problems.py

```diff
@@ -93,21 +93,12 @@ def submit_problem(step):
     # first scroll down so the loading mathjax button does not
     # cover up the Submit button
     world.browser.execute_script("window.scrollTo(0,1024)")
-    assert world.is_css_not_present("button.submit.is-disabled")
```

Comment:
I don't care if the button has a class on it or not. If the button is disabled, clicking on it won't do anything.